editor/src/uuiui/icons.tsx

```diff
@@ -70,6 +70,79 @@ export const LargerIcons = {
   }),
 }
 
+export const InspectorSectionIcons = {
+  Layout: makeIcon({
+    category: 'inspector',
+    type: 'layout',
+    color: 'main',
+    width: 16,
+    height: 16,
+  }),
+  LayoutSystem: makeIcon({
+    category: 'inspector',
+    type: 'layout-system',
+    color: 'main',
+    width: 16,
+    height: 16,
+  }),
+  Layer: makeIcon({
+    category: 'inspector',
+    type: 'layer',
+    color: 'main',
+    width: 16,
+    height: 16,
+  }),
+  Background: makeIcon({
+    category: 'inspector',
+    type: 'background',
+    color: 'main',
+    width: 16,
+    height: 16,
+  }),
+  Border: makeIcon({
+    category: 'inspector',
+    type: 'layout',
```

Comment:
This would be the reason for the incorrect border icon

Reply:
🧐

Reply:
how peculiar!